docs/products/clickhouse/howto/connect-with-jdbc.rst

```diff
@@ -1,9 +1,9 @@
-Connect Aiven for ClickHouse® to an external databases via JDBC
+Connect Aiven for ClickHouse® to external databases via JDBC
 ===============================================================
 
 You can use `ClickHouse JDBC driver <https://github.com/ClickHouse/clickhouse-jdbc/tree/master/clickhouse-jdbc>`_ to connect external sources to your Aiven for ClickHouse database.
 
-You will need Aiven for ClickHouse service, accessible by HTTPS. The connection values you'll need can be found in your Aiven for ClickHouse service page, in the connection information for *HTTPS endpoint*.
+You will need Aiven for ClickHouse® service, accessible by HTTPS. The connection values you'll need can be found in your Aiven for ClickHouse® service page in the connection information for *HTTPS endpoint*.
```

Comment:
You can always have one (R) per product per article :) More is not harmful, but only one is required by legal

Reply:
ha! good to know - that helps :-)